aten/src/ATen/native/QuantizedLinear.cpp

```diff
@@ -450,19 +454,30 @@ std::tuple<Tensor, Tensor, double, int64_t> fbgemm_linear_quantize_weight(
       false, "This PyTorch installation was not built with FBGEMM operators");
 }
 
+Tensor fbgemm_pack_quantized_matrix(const Tensor& /*input*/) {
+  // We make a strong guarantee that models using these operators will have the
+  // same numerics across different machines. Therefore, we do not provide a
+  // fallback path and rather fail loudly if we cannot run FBGEMM.
+  TORCH_CHECK(
+      false, "This PyTorch installation was not built with FBGEMM operators");
+}
+
 Tensor fbgemm_pack_quantized_matrix(
     const Tensor& /*input*/,
     int64_t /*K*/,
     int64_t /*N*/) {
+  TORCH_WARN(
```

Comment:
nit: Since this function just calls another function - you can pull out this wrapper outside of #ifdef